src/actions/auth_actions/confirm_login_action.js

```diff
@@ -12,8 +14,9 @@ Actions.register(CONFIRM_LOG_IN, payload => {
   })
   .then(checkStatus)
   .then(response => {
-    authStore.setAuthed(response.data.token);
-    authStore.setIsUserNew(response.data.is_new);
+    googleAuthStore.setAuthed(response.data.token);
+    googleAuthStore.setIsUserNew(response.data.is_new);
+    Cookies.set('service', GOOGLE_AUTH_SERVICE, { expires: 1 });
```

Comment:
Curious about this expiration, as well. Is that 1 day? Wondering how we decide to set this and what happens if it's expired vs not when someone visits the homepage url

Reply:
yeah, it's 1 day. we could always change when these expire too in the future